static/src/js/components/Map/LayerSwitcherControl.jsx

```diff
@@ -0,0 +1,131 @@
+import ReactDOM from 'react-dom'
+import Control from 'ol/control/Control'
+import EventType from 'ol/events/EventType'
+import mapLayers from '../../constants/mapLayers'
+
+// Define base layers outside of the class
+const baseLayers = [
+  mapLayers.worldImagery,
+  mapLayers.correctedReflectance,
+  mapLayers.landWaterMap
+]
+
+/**
+ * This class adds a layer switcher control to the map
+ */
+class LayerSwitcherControl extends Control {
+  constructor(options) {
+    const element = document.createElement('div')
+    element.className = options.className
+
+    super({
+      ...options,
+      element
+    })
+
+    this.options = options
+    this.onChangeLayer = options.onChangeLayer || (() => {})
+
+    const toggleButton = document.createElement('button')
+    toggleButton.className = `${options.className}__button`
+    toggleButton.setAttribute('type', 'button')
+    toggleButton.title = 'Layer Options'
+    toggleButton.ariaLabel = 'Layer Options'
+
+    if (options.LayersIcon) {
+      ReactDOM.render(
+        options.LayersIcon,
+        toggleButton
+      )
+    }
+
+    element.appendChild(toggleButton)
+
+    const panel = document.createElement('div')
+    panel.className = `${options.className}__panel`
+
+    this.addLayerOptions(panel)
+    element.appendChild(panel)
+
+    element.addEventListener('mouseenter', () => {
+      panel.classList.add(`${options.className}__panel--visible`)
+    })
+
+    element.addEventListener('mouseleave', () => {
+      panel.classList.remove(`${options.className}__panel--visible`)
+    })
+  }
+
+  /**
+   * Adds layer options to the panel
+   */
+  addLayerOptions(panel) {
+    const { className } = this.options
+    const layerOptions = this.options.layerOptions || [
```

Comment:
This default value needs to be removed, you are defining this array of options here and in MapControls.js